router/jsonrpc_test.go

```diff
@@ -16,17 +16,25 @@ import (
 
 	"github.com/AccumulateNetwork/accumulated/internal/relay"
 	acctesting "github.com/AccumulateNetwork/accumulated/internal/testing"
+	"github.com/AccumulateNetwork/accumulated/networks"
 	"github.com/AccumulateNetwork/accumulated/types"
 	"github.com/AccumulateNetwork/accumulated/types/api"
 	"github.com/go-playground/validator/v10"
 )
 
-var testnet = flag.Int("testnet", 4, "TestNet to load test")
+var testnet = flag.String("testnet", "Localhost", "TestNet to load test")
 var loadWalletCount = flag.Int("loadtest-wallet-count", 100, "Number of wallets")
 var loadTxCount = flag.Int("loadtest-tx-count", 1000, "Number of transactions")
 
-func _TestLoadOnRemote(t *testing.T) {
-	txBouncer := relay.NewWithNetworks(*testnet)
+func TestLoadOnRemote(t *testing.T) {
+	if os.Getenv("CI") == "true" {
+		t.Skip("This test is not appropriate for CI")
+	}
```

Comment:
This test will be skipped when run from CI. This test either requires connecting to a remote IP or running a local node. Neither are appropriate for CI unit tests. If we want CI integration tests, we should set those up explicitly as integration tests.